fabric/main.py

```diff
@@ -125,6 +126,7 @@ def is_classic_task(tup):
             callable(func)
             and (func not in _internals)
             and not name.startswith('_')
+            and not (inspect.isclass(func) and issubclass(func, Exception))
```

Comment:
I'm wondering to myself if it makes sense that any class is a task? I'm wondering if maybe we should just refuse classes. Suppose I want to generate some csv from a bunch of servers and I put the following in my file: `from csv import CsvWriter, Dialect` -- with current master, fabric will say `    Dialect     Describe an Excel dialect.` in `fab -l`.

Maybe we shouldn't remove only exceptions here?